app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -17,1024 +17,903 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.7.0\n"
+"Generated-By: Babel 2.8.0\n"
 
-#: app/templates/404.html:14
+#: app/templates/404.html:13
 msgid "Page not found"
 msgstr "Heb ddod o hyd i'r dudalen"
 
-#: app/templates/404.html:16
+#: app/templates/404.html:15
 msgid "If you entered a web address, check it is correct."
 msgstr "Os gwnaethoch nodi cyfeiriad gwefan, gwnewch yn siŵr ei fod yn gywir."
 
-#: app/templates/404.html:17
+#: app/templates/404.html:16
 msgid "If you pasted the web address, check that you copied the entire address."
 msgstr ""
 "Os gwnaethoch ludo cyfeiriad y wefan, gwnewch yn siŵr eich bod wedi "
 "copïo'r cyfeiriad cyfan."
 
-#: app/templates/404.html:19
+#: app/templates/404.html:18
 #, python-format
 msgid ""
 "If you still encounter problems please %(open)scontact us%(close)s. We "
 "apologise for any inconvenience this may have caused."
 msgstr "Os ydych chi'n cael problemau o hyd, %(open)scysylltwch â ni%(close)s."
 
-#: app/templates/accessibility.html:20 app/templates/accessibility.html:27
-msgid "Accessibility"
-msgstr "Hygyrchedd"
+#: app/templates/closed.html:12
+msgid "This survey has closed"
+msgstr "Mae'r arolwg hwn wedi cau"
 
-#: app/templates/accessibility.html:24
-msgid "Home"
-msgstr "Hafan"
+#: app/templates/closed.html:15
+msgid "Your access code is for a survey that has now closed."
+msgstr "Mae eich cod mynediad ar gyfer arolwg sydd bellach wedi cau."
 
-#: app/templates/accessibility.html:34
-msgid "Census questionnaire accessibility statement"
-msgstr "Datganiad hygyrchedd holiadur y cyfrifiad"
+#: app/templates/closed.html:17
+msgid "Thank you for your help."
+msgstr "Diolch am eich help."
 
-#: app/templates/accessibility.html:36
-msgid ""
-"This census is run by the Office for National Statistics. We want as many"
-" people as possible to be able to do the census online. For example, that"
-" means you should be able to:"
+#: app/templates/common-address-in-scotland.html:13
+msgid "This address is not part of the census for England and Wales"
 msgstr ""
-"Cynhelir y cyfrifiad hwn gan y Swyddfa Ystadegau Gwladol. Rydym am i "
-"gynifer o bobl â phosibl allu gwneud y cyfrifiad ar lein. Er enghraifft, "
-"mae hynny'n golygu y dylech fod yn gallu:"
 
-#: app/templates/accessibility.html:39
-msgid "change colours, contrast levels and fonts"
-msgstr "newid lliwiau, lefelau cyferbyniad a ffontiau"
+#: app/templates/common-address-in-scotland.html:15
+msgid ""
+"You have selected an address in Scotland. Visit the <a "
+"href=\"https://www.scotlandscensus.gov.uk/\">National Records of Scotland"
+" website</a> to find out how to take part in the census for Scotland."
```

Comment:
```suggestion
" website</a> to find out how to take part in the census for Scotland next year."
```
Just a suggestion - otherwise people might think there's a Scottish Census going on at the same time.